graphql/resolvers.go

```diff
@@ -1160,26 +1160,14 @@ func (r *queryResolver) PatchTasks(ctx context.Context, patchID string, sorts []
 }
 
 func (r *queryResolver) TaskTests(ctx context.Context, taskID string, execution *int, sortCategory *TestSortCategory, sortDirection *SortDirection, page *int, limit *int, testName *string, statuses []string) (*TaskTestResult, error) {
-	dbTask, err := task.FindByIdExecution(taskID, execution)
-	if dbTask == nil || err != nil {
-		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("cannot find task with id %s", taskID))
-	}
-	baseTask, err := dbTask.FindTaskOnBaseCommit()
-	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding base task with id %s: %s", taskID, err))
-	}
-
-	var taskExecution int
-	taskExecution = dbTask.Execution
-
-	baseTestStatusMap := make(map[string]string)
-	if baseTask != nil {
-		baseTestResults, _ := r.sc.FindTestsByTaskId(baseTask.Id, "", "", "", 0, taskExecution)
-		for _, t := range baseTestResults {
-			baseTestStatusMap[t.TestFile] = t.Status
-		}
-	}
-
+	opts := apimodels.GetCedarTestResultsOptions{
+		BaseURL:   evergreen.GetEnvironment().Settings().Cedar.BaseURL,
+		TaskID:    taskID,
+		Execution: *execution,
+	}
+	cedarTestResults, _ := apimodels.GetCedarTestResults(ctx, opts)
```

Comment:
do you know if the CedarTestResult struct should contain a GroupId? GroupId is available from the TestResult struct which represents testresults queried from evergreen.

Reply:
I'm not sure I don't see that. https://github.com/evergreen-ci/evergreen/blob/01f29a346d16d5045f07a8494906088a049b481a/model/testresult/testresult.go#L25-L56

Reply:
GroupId was added to TestResult in these code changes: https://github.com/evergreen-ci/evergreen/pull/4458/files#diff-860d85041044ecc18c5eb5915d87ab3b1942b9271bf51585c683bf0cdb2884b8R20

Reply:
The options struct does not need group id, but the individual test results returned will have that field.